package.json

```diff
@@ -19,14 +19,14 @@
   },
   "devDependencies": {
     "aegir": "^20.4.1",
-    "async-iterator-all": "^1.0.0",
     "chai": "^4.2.0",
     "cids": "^0.7.1",
     "go-ipfs-dep": "~0.4.22",
     "ipfsd-ctl": "~0.47.4",
     "peer-id": "~0.13.5"
   },
   "dependencies": {
+    "async-iterator-all": "^1.0.0",
```

Comment:
PSA this module has been renamed https://www.npmjs.com/package/it-all

Reply:
At least it has a better name. I wll update the usage of this module when I have some time. Thanks for letting us know Alan